core/config/config.go

```diff
@@ -99,6 +106,13 @@ func Load(ctx context.Context, db pg.DB, sdb *sinkdb.DB) (*Config, error) {
 	return c, nil
 }
 
+func idMatchesPG(ctx context.Context, id string, db pg.DB) bool {
+	const q = `SELECT id FROM core_id`
+	var pgID string
+	err := db.QueryRowContext(ctx, q).Scan(&pgID)
+	return err == nil && pgID == id
+}
```

Comment:
If there's an error querying the database here, we'll end up returning ErrStaleRaftConfig from Load which is potentially inaccurate. Maybe we should add an error return value to differentiate between trouble connecting to the database and a wiped Postgres database.

Heads up that If there is no Core ID in the table, QueryRowContext will return [sql.ErrNoRows](https://godoc.org/database/sql#pkg-variables). idMatchesPG could translate that specific error into a `(false, nil)` return value and return any other error as-is.